### PR TITLE
tests(ticdc): refine the default-value integration test to save running time

### PR DESCRIPTION
**Pull request by maxshuang**
### What problem does this PR solve?

Issue Number: close  https://github.com/pingcap/tiflow/issues/4785

Run multi-tests parallel as possible and reduce DDL nums.

Aim to reduce the cost time from 35 min+ to 15 min+

```release-note
 `None`.
```

### CODE REVIEW

**Comment by maxshuang** [comment]
> /run-dm-compatibility-test

Pls don't run any test, I am still working on it.
Too many DDL will cost too much time.

**Comment by hi-rustin** [comment]
I see that the current run time is still very long. 10:58 - 11:20

**Review by sdojjy — APPROVED** [review]

File: tests/integration_tests/default_value/main.go [hunk 1]
```diff
@@ -65,52 +68,62 @@ func main() {
...
+func testGetDefaultValue(srcs []*sql.DB, wg *sync.WaitGroup) {
+	defer wg.Done()
 	start := time.Now()
 	defer func() {
 		log.S().Infof("testGetDefaultValue take %v", time.Since(start))
 	}()
 
-	var total int
+	var wg2 sync.WaitGroup
 	for i, ddlFunc := range []func(context.Context, *sql.DB){
```
Comment (CharlesCheung96) [inline tests/integration_tests/default_value/main.go:92]:
```suggestion
	ddlFuncs := []func(context.Context, *sql.DB){
		modifyColumnDefaultValueDDL1, modifyColumnDefaultValueDDL2,
	}
	for i, ddlFunc := range ddlFuncs
```

Comment (maxshuang) [reply tests/integration_tests/default_value/main.go:92]: Seems no big difference.
I prefer keeping the previous one.

**Review by CharlesCheung96 — APPROVED** [review]

File: tests/integration_tests/default_value/main.go [hunk 2]
```diff
@@ -273,420 +288,584 @@ func modifyColumnDefaultValueDDL2(ctx context.Context, db *sql.DB) {
...
-		{
-			"alter table test.`%s` add column v1 float(5,2) unsigned not null",
-			13.13,
-		},
-		{
-			"alter table test.`%s` add column v1 double(5,2) unsigned not null",
-			13.13,
-		},
-
+		/*
```
Comment (hi-rustin) [inline tests/integration_tests/default_value/main.go:695]: Can you add some comments as to why you are commenting them out? Or we could just remove it?

**Review by asddongmen — APPROVED** [review]

File: tests/integration_tests/default_value/main.go (hunk 2 above)
Comment (maxshuang) [reply tests/integration_tests/default_value/main.go:695]: I already add some comments at the begining.

**Pull request merged**